zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/configuration/NetworkCfg.java

```diff
@@ -13,6 +13,7 @@
 import java.time.Duration;
 import org.springframework.util.unit.DataSize;
 
+// BROKER-ONLY
```

Comment:
:wrench: I would suggest to make it more descriptive - Network configuration for broker

Reply:
This change is not necessary actually, I removed it.